Documentation/installation/osx/install.md

```diff
@@ -18,11 +18,12 @@ $ go get -u github.com/derekparker/delve/cmd/dlv
 
 With this method you will not be able to use delve's native backend.
 
-Alternatively, you can clone the repo and run:
+Alternatively, you can clone the repo into `$GOPATH/github.com/derekparker/delve` and run:
```

Comment:
Isn’t this path missing the `/src/` path component?

Reply:
Yes. My mistake.